packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1894,17 +1778,31 @@ function commitResetTextContent(current: Fiber): void {
   resetTextContent(current.stateNode);
 }
 
-function commitPassiveWork(finishedWork: Fiber): void {
+function commitPassiveUnmountInsideDeletedTree(finishedWork: Fiber): void {
```

Comment:
I guess I missed this, but this name looks like the opposite of what it should be.

`commitPassiveUnmount ` should be the function renamed to `commitPassiveUnmountInsideDeletedTree` – since it requires a `nearestMountedAncestor`.

I'll update this as part of #19895.